src/index.js

```diff
@@ -0,0 +1,155 @@
+import 'lodash';
+
+// SELECT ELEMENTS
+const form = document.getElementById('todoform');
+const todoInput = document.getElementById('newtodo');
+const todosListEl = document.getElementById('todos-list');
+const notificationEl = document.querySelector('.notification');
+// deleteselecteditems
+const deleteButton = document.getElementById('removeitem');
+
+// VARS
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+let EditTodoId = -1;
+
+// RENDER TODOS
+function renderTodos() {
+  if (todos.length === 0) {
+    todosListEl.innerHTML = '<center>Add tasks here and Press Enter !</center>';
+    return;
+  }
+
+  // CLEAR ELEMENT BEFORE A RE-RENDER
+  todosListEl.innerHTML = '';
+
+  // RENDER TODOS
+  todos.forEach((todo, index) => {
+    todosListEl.innerHTML += `
+    <div class="todo" id=${index}>
+      <i 
+        class="bi ${todo.completed ? 'bi-check-circle-fill' : 'bi-circle'}"
+        style="color : ${todo.color}"
+        data-action="check"
+      ></i>
+      <p class="${todo.completed ? 'completed' : ''}"  onclick="this.classList.toggle('selected')" data-action="check" >${todo.value}</p>
+      <i class="bi bi-pencil-square" data-action="edit"></i>
+      <i class="bi bi-trash" data-action="delete"></i>
+    </div>
+    `;
+  });
+}
+
+// CHECK A TODO
+function checkTodo(todoId) {
+  todos = todos.map((todo, index) => ({
+    ...todo,
+    completed: index === todoId ? !todo.completed : todo.completed,
+  }));
+
+  renderTodos();
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
```

Comment:
- [ Optional ] As stated the project requirement kindly  import `checkTodo` function as module which  contain methods related to the status updates (completed: true / false).

![Capture](https://user-images.githubusercontent.com/93598189/206875997-f70612f7-dad9-4010-a7d0-0eb90a0a0ad9.PNG)